Controller/PostController.php

```diff
@@ -30,16 +31,17 @@ public function homeAction()
     }
 
     /**
-     * @param array $criteria
-     * @param array $parameters
+     * @param array   $criteria
+     * @param array   $parameters
+     * @param Request $request
      *
      * @return \Symfony\Component\HttpFoundation\Response
      */
-    public function renderArchive(array $criteria = array(), array $parameters = array())
+    public function renderArchive(array $criteria = array(), array $parameters = array(), Request $request)
```

Comment:
This BC break.

Please see `CRUDController` from admin-bundle to see how this is handled.